BackEnd/src/middleware.php

```diff
@@ -10,4 +10,11 @@
     'users' => $slim->getContainer()->get('authorizedUser')
 ]));
 
+$slim->add(new HttpBasicAuthentication([
+    'relaxed' => [],
+    'path' => '/deleteProject',
```

Comment:
You misconfigured the middleware, no need to add a second instance.

See https://github.com/tuupola/slim-basic-auth for more information